crates/polars-sql/Cargo.toml

```diff
@@ -16,7 +16,8 @@ polars-plan = { version = "0.32.0", path = "../polars-plan", features = ["compil
 
 serde = { workspace = true }
 serde_json = { workspace = true }
-sqlparser = { workspace = true }
+sqlparser = { path = "../../../../sqlparser-rs" }
```

Comment:
Please don't commit this.

Reply:
As per my previous comment, this is blocked by an issue in the `sqlparser` I'm rerouting here. This exists exclusively to test the `polars` fix after there's a patch for that issue. I'll revert the reroute when this leaves draft.